src/main/java/org/gradle/profiler/asyncprofiler/AsyncProfilerOutputType.java

```diff
@@ -9,13 +9,13 @@
     STACKS("collapsed") {
         @Override
         File outputFileFor(ScenarioSettings settings) {
-            return settings.profilerOutputLocationFor(".stacks.txt");
+            return settings.profilerOutputLocationFor(".stacks.txt").getAbsoluteFile();
```

Comment:
I wonder if we should go even deeper: It seems like `ScenarioDefinition.getOutputDir()` or `ScenarioDefinition.outputDir` should always be an absolute file. Then the rest would follow automatically I think.

Reply:
We can do that